### PR TITLE
Setup the static site for staging and the lambda for prod

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Add workflows to setup the system_status lambda for production and the system_status_static_site for staging

### CODE REVIEW

File: .github/workflows/merge_to_main_staging.yml [hunk 1]
```diff
@@ -173,6 +173,11 @@ jobs:
           cd env/staging/system_status
           terragrunt apply --terragrunt-non-interactive -auto-approve
 
+      - name: Apply aws/system_status_static_site
+        run: |
+          cd env/staging/system_status_static_site
```
Comment (ben851) [inline .github/workflows/merge_to_main_staging.yml:178]: This folder doesn't seem to exist?

**Review by ben851 — CHANGES_REQUESTED** [review]
Seems that we're missing the static site env folder and config... we'll need to add that or else the pipeline will crash

File: .github/workflows/merge_to_main_staging.yml (hunk 1 above)
Comment (jzbahrai) [reply .github/workflows/merge_to_main_staging.yml:178]: i totally forgot to add the terragrunt hcl files! thanks

File: .github/workflows/terragrunt_plan_staging.yml [hunk 2]
```diff
@@ -324,3 +327,13 @@ jobs:
...
+
+      - name: Terragrunt plan aws/system_status_static_site
+        if: ${{ steps.filter.outputs.system_status_static_site == 'true' || steps.filter.outputs.common == 'true' }}
+        uses: cds-snc/terraform-plan@v3
+        with:
+          directory: "env/staging/system_status_static_site"
+          comment-delete: "true"
+          comment-title: "Staging: system_status_static_site"
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          terragrunt: "true"
```
Comment (ben851) [inline .github/workflows/terragrunt_plan_staging.yml:339]:
```suggestion
          terragrunt: "true"
          skip-conftest: "true"
```

File: .github/workflows/terragrunt_plan_production.yml [hunk 3]
```diff
@@ -207,3 +211,12 @@ jobs:
...
           terragrunt: "true"
+
+      - name: Terragrunt plan system_status
+        uses: cds-snc/terraform-plan@v3
+        with:
+          directory: "env/production/system_status"
+          comment-delete: "true"
+          comment-title: "Production: system_status"
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          terragrunt: "true"
```
Comment (ben851) [inline .github/workflows/terragrunt_plan_production.yml:222]:
```suggestion
          terragrunt: "true"
          skip-conftest: "true"
```

**Review by ben851 — APPROVED** [review]

**Pull request merged**